pkgs/development/python-modules/debtcollector/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pbr
+, six
+, wrapt
+}:
+let 
+  pyModuleDeps = [
+    pbr
+    six
+    wrapt
+  ];
+in
+buildPythonPackage rec {
+  pname = "debtcollector";
+  version = "2.2.0";
+
+  src = fetchPypi {
+    pname = "debtcollector";
```

Comment:
inherit pname

Reply:
I like having pname hard-written in fetchers because sometimes the name doesn't match the derivation pname. I also find it easier to read instead of constantly having to search for the values above. But if this a convention, I'm happy to follow it.

Reply:
Convention is indeed to inherit.

Reply:
Fixed.